core/src/main/resources/schema/infinispan-config-8.1.xsd

```diff
@@ -535,7 +535,27 @@
     </xs:attribute>
   </xs:complexType>
   
-  <xs:complexType name="state-persistence">
+  <xs:complexType name="global-state">
+    <xs:sequence>
+      <xs:element name="persistent-location" type="tns:global-state-path" minOccurs="1" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation>
+            Defines the path where global state for this cache-container will be stored. The data stored at this location is required for graceful
+            shutdown and restore.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="swap-location" type="tns:global-state-path" minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation>
+            Defines the path which will be used to store temporary swap data.
```

Comment:
actually it's not on /tmp the default, but in the current directory. I think if the user opts to store the indexes on disk, they are supposed to be durable, re-generating it involves running the massindexer and it can be a long running process